content/terraform/v1.13.x/docs/language/meta-arguments/lifecycle.mdx

```diff
@@ -0,0 +1,329 @@
+`update` operation. The arguments are the relative address of the attributes in the resource. You can reference map and list elements using index notation,
+such as `tags["Name"]` and `list[0]`.
+
+In the following example, Terraform ignores changes to `tags` so that a management agent can update them based on a rule set managed elsewhere:
+
+```hcl
+resource "aws_instance" "example" {
+   # ...
+   lifecycle {
+      ignore_changes = [
+         tags
+      ]  
+   }
+}
+```
+
+Instead of a list of items, you can use the `all` keyword to instruct
+Terraform to ignore all attributes. As a result, Terraform can
+create and destroy the remote object but will never propose updates to it.
+
+Terraform only ignores attributes defined by the resource type. You can't apply 
+`ignore_changes` to itself or to any other meta-arguments.
+
+You can use `ignore_changes` in [`resource`](/terraform/language/block/resource) blocks. 
+
+### `replace_triggered_by` 
+
+Terraform replaces the resource when any of the referenced resources or specified attributes change. Supply a list of expressions that reference managed resources, instances, or instance attributes. 
+
+When used in a resource that uses [`count`](/terraform/language/meta-arguments/count) or [`for_each`](/terraform/language/meta-arguments/for_each), you can use `count.index` or `each.key` in the expression to reference specific instances of other resources that are configured with the same count or collection.
+
+References trigger replacement in the following conditions:
+
+- If the reference is to a resource with multiple instances, a plan to
+   update or replace any instance triggers a replacement.
+- If the reference is to a single resource instance, a plan to update or
+   replace that instance triggers a replacement.
+- If the reference is to a single attribute of a resource instance, any
+   change to the attribute value triggers a replacement.
+
+You can only reference managed resources in `replace_triggered_by`
+expressions. This lets you modify these expressions without forcing
+replacement. In the following example, Terraform replaces `aws_appautoscaling_target` each time this instance of `aws_ecs_service` is replaced:
+
+
+```hcl
+resource "aws_appautoscaling_target" "ecs_target" {
+   # ...
+   lifecycle {
+      replace_triggered_by = [
+         aws_ecs_service.svc.id
+      ]
+   }
+}
+```
+
+`replace_triggered_by` allows only resource addresses because the decision is based on the planned actions for all of the given resources. Plain values, such as local values or input variables, do not have planned actions of their own, but you can treat them with a resource-like lifecycle by using them with the [`terraform_data` resource](/terraform/language/resources/terraform-data).
+
+You can use `replace_triggered_by` in [`resource`](/terraform/language/block/resource) blocks. 
+
+### `precondition` 
```

Comment:
Many of the other H3s have examples but pre and post conditions don't. I do see there are examples in the examples section, so this isn't a blocker, just inconsistent.